demos/storybook/package.json

```diff
@@ -10,7 +10,8 @@
         "lint": "eslint \"stories/**/**.ts\"",
         "lint:fix": "eslint \"stories/**/**.ts\" --fix",
         "prettier": "prettier \"**/**.{ts,json,css,scss,html}\" --write",
-        "prettier:check": "prettier \"**/**.{ts,json,css,scss,html}\" --check"
+        "prettier:check": "prettier \"**/**.{ts,json,css,scss,html}\" --check",
+        "deploy:feature": "yarn add firebase-tools && ./node_modules/.bin/firebase deploy --non-interactive --token $FIREBASE_FEATURE_TOKEN"
```

Comment:
Do we need to have yarn add do a fresh install every time in the command, or should we add firebase-tools to our devDependencies so it can be cached with the rest of our dependencies? Or use npx?

Reply:
Yes we can add the firebase tools to our devDependencies list so that we can cache this dependency.